includes/classes/blocks/class-template-parts.php

```
@@ -206,24 +206,47 @@ protected function maybe_create_template_part( $slug, $args, $theme ) {
 		}
 
 		// Check if template part already exists.
-		$existing = get_posts(
+		// Note: We must use tax_query WITHOUT the 'name' parameter because
+		// WordPress ignores tax_query when 'name' is present in the query args.
+		$query = new \WP_Query(
```

PHPMD (2.15.0)

Missing class import via use statement (line '211', column '16'). (undefined)

(MissingImport)